package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@livelyvideo/mediasoup",
-  "version": "3.12.12-lv3-notranscode",
+  "version": "3.12.15-lv3-notranscode-PM-2288",
```

Comment:
If this code change based on versatica's 3.12.15? If not, then it should probably be 3.12.12-lv4-... so that we can tell that which versatica's baseline we last picked into our branch. Or let's come up with some other way to keep track of this number.

Reply:
oh, I see good point, I will fix that. thanks